config/overlays/helm/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+bases:
+  - ../../default
+
+namespace: |-
+  {{ include "chart.namespace" . }}
+
+patchesStrategicMerge:
```

Comment:
I'm having a tough time understanding this.

Reply:
Ive added comments to describe this behaviour